packages/flutter_template_images/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
Without a version bump this won't be usable.

Reply:
Done!